main.js

```diff
@@ -0,0 +1,90 @@
+//'use strict';
+const body = document.querySelector('body');
+const nameInput = document.querySelector('.name');
+const phoneNumber = document.querySelector('.number');
+const submitButton = document.querySelector('.submit');
+const nameValidationRE = /^([a-z]*\s[a-z]*((\s([a-z]*))?)$)/i;
+const numberValidationRE = /(\+380)?(380)?0[- ]?\d\d[- ]?\d\d\d[- ]?\d\d[- ]?\d\d$/;
+const checkboxInfo = document.querySelector('.checkbox');
+const phoneRegionInfo = document.querySelector('.phone-region');
+const regionSelection = document.querySelector('.region');
+
+nameInput.addEventListener('change', validateName);
+function validateName() {
+  let valueName = nameInput.value.trim();
+  if (!nameValidationRE.test(valueName) ) {
```

Comment:
what is this `test`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test